app/app.go

```diff
@@ -1329,8 +1329,9 @@ func (app *App) ProcessBlock(ctx sdk.Context, txs [][]byte, req BlockProcessRequ
 	midBlockEvents := app.MidBlock(ctx, req.GetHeight())
 	events = append(events, midBlockEvents...)
 
-	// run other txs
-	otherResults, ctx := app.BuildDependenciesAndRunTxs(ctx, txs)
+	// run other txs - this will run them synchronously without building a dag or executing concurrently
+	// otherResults, ctx := app.BuildDependenciesAndRunTxs(ctx, txs)
+	otherResults := app.ProcessBlockSynchronous(ctx, txs)
```

Comment:
could we leave a comment here or in the pr description around the plans for re-enabling it?

Reply:
yup, will do